migrations/14_SCHEMES_SchemeRegistrar.js

```diff
@@ -1,24 +1,24 @@
 /* global artifacts, web3 */
 /* eslint no-undef: "error" */
-
+const assert = require('assert')
 const { getGenesisProtocolData } = require('../src/helpers/genesisProtocolHelper')(artifacts)
 
 const DxAvatar = artifacts.require('DxAvatar')
 const DxController = artifacts.require('DxController')
 
-const registerScheme = require('./helpers/registerScheme')
-const { SchemePermissions: {
+const { registerScheme, setParameters, SchemePermissions } = require('./helpers/schemeUtils')
+const {
   REGISTERED,
   REGISTER_SCHEMES,
   ADD_REMOVE_GLOBAL_CONSTRAINTS,
   UPGRADE_CONTROLLER,
   CALL_DELEGATECALL
-} } = registerScheme
+} = SchemePermissions
 
 const getDaostackContract = require('../src/helpers/getDaostackContract')(web3, artifacts)
 
 
-module.exports = async function (deployer) {
+module.exports = async function (deployer, network) {
```

Comment:
Why add `network` if you don't use it?

Reply:
Network is not required, but extrangely it requires deployer, even if you don't use it. I think truffle 5 does a check somehow. 

If you remove the deployer, the migrations fail.